e2e_tests/test_browse.py

```diff
@@ -0,0 +1,92 @@
+import time
+
+from playwright.sync_api import Page
+
+
+def test_has_title(page: Page):
+    time.sleep(1)
```

Comment:
shouldnt need sleep in the tests themselves. I think we can just run the playwright with `-slowmo=200` for example to pause 200ms between calls